blog.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
```

Comment:
You'll want to add the meta tag that's referenced in week 1 to all of your HTML files. It'll make your life easier when it's time for styling.